src/iotjs_env.c

```diff
@@ -122,7 +122,7 @@ bool iotjs_environment_parse_command_line_arguments(iotjs_environment_t* env,
       size_t port_length = sizeof(strlen(argv[i] - port_arg_len - 1));
       char port[port_length];
       memcpy(&port, argv[i] + port_arg_len, port_length);
-      sscanf(port, "%d", &(_this->config.debugger->port));
+      sscanf(port, "%hd", &(_this->config.debugger->port));
```

Comment:
What about %hu?

Reply:
Yeah, that'd be even better, thanks. Patch updated.